### PR TITLE
Time limit overrides for extensions

### PR DESCRIPTION
**Pull request by przemyslaw-dobrowolski-cl**
This commit adds a configuration option to setup orverrides
for the default time limit for running an extension. A single
setting consists of a regex path, a regex event name and
a time duration.

https://github.com/cloudwan/gohan/issues/361

### CODE REVIEW

File: extension/otto/otto.go [hunk 1]
```diff
@@ -128,10 +129,15 @@ func (env *Environment) LoadExtensionsForPath(extensions []*schema.Extension, pa
 			if err != nil {
 				return err
 			}
-
 		}
 	}
-
+	// setup time limits for matching extensions
```
Comment (p-kozlowski) [inline extension/otto/otto.go]: code duplicated with extension/gohanscript/env.go

File: etc/gohan.yaml [hunk 2]
```diff
@@ -67,4 +67,13 @@ extension:
...
+  - go
+
+  # default time limit in secs for running extension
+  timelimit: 10
+
+  # time limit overrides in secs for running particular extension in path
+  timelimits:
+  - path: /v1.0/store/orders
+    event: pre_*
+    timelimit: 1
```
Comment (p-kozlowski) [inline etc/gohan.yaml]: without looking through the source code it's hard to guess what is the unit here. timelimit_seconds?

Comment (przemyslaw-dobrowolski-cl) [inline etc/gohan.yaml]: If it wouldn't be for the consistency with the existing field 'timelimit' I'd change it according to your suggestion. IMO, its better to copy the problematic naming in this case.

**Comment by przemyslaw-dobrowolski-cl** [comment]
@nati  a kind remainder about this PR

**Pull request merged**